### PR TITLE
Feat: Readme Updates

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description
An update to the boss room readme that matches with the refresh of the [bitesize one](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/96)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [ ] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] JIRA ticket ID is in the PR title or at least one commit message
 - [ ] Include the ticket ID number within the body message of the PR to create a hyperlink
 - [ ] An Index entry has been added in readme.md if applicable

### CODE REVIEW

**Review by MorwennaG — DISMISSED** [review]
This looks _great_, Jil. Could you update the templates to match?

**Review by fernando-cortez — COMMENTED** [review]
The first two badges (UnityVersion & NetcodeVersion) are missing links. Add that and it's gtg!

File: README.md [hunk 1]
```diff
@@ -1,86 +1,113 @@
 ![Banner](Documentation/Images/Banner.png)
-## Co-op multiplayer RPG and utilities built with Unity Netcode for GameObjects
+<br><br>
+
+# Boss Room: a Co-op, Multiplayer RPG Sample
+###  Made with and Including Utilities for Netcode for GameObjects
+<br>
+
+[![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)](https://unity.com/releases/editor/qa/lts-releases#:~:text=February%2014%2C%202023-,LTS%20Release,2021.3.18f1,-Released%3A%20February)
```
Comment (fernando-cortez) [inline README.md:8]: Boss Room is still on 2021.3.15, so the readme should point to that version for the time being.

**Comment by fernando-cortez** [comment]
Something I found after my review: I was testing the header links and some of them don't direct to the right spot in the readme.

File: README.md [hunk 2]
```diff
@@ -1,86 +1,113 @@
...
+- ### [Index of resources in this project](#index-of-resources-in-this-project)
   - [Gameplay](#gameplay)
   - [Game Flow](#game-flow)
   - [Connectivity](#connectivity)
   - [Services (Lobby, Relay, etc)](#services-lobby-relay-etc)
   - [Tools and Utilities](#tools-and-utilities)
-- [Troubleshooting](#troubleshooting)
-- [Licence](#licence)
-- [Other samples](#other-samples)
+- ### [Troubleshooting](#troubleshooting)
```
Comment (fernando-cortez) [inline README.md:66]: I was testing the header links and some of them don't direct to the right spot in the readme. This line is one example.

File: README.md [hunk 3]
```diff
@@ -1,86 +1,113 @@
...
  - __Alternatively:__ click the green `Code` button and then select the 'Download Zip' option.  Please note that this will download the branch that you are currently viewing on Github.  
  - __Windows users:__ Using Windows' built-in extraction tool may generate an "Error 0x80010135: Path too long" error window which can invalidate the extraction process. A workaround for this is to shorten the zip file to a single character (eg. "c.zip") and move it to the shortest path on your computer (most often right at C:\\) and retry. If that solution fails, another workaround is to extract the downloaded zip file using [7zip](https://www.7-zip.org/).
+<br><br>
+
+## Requirements
 
+BossRoom is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation.
+
+**PLEASE NOTE:** You will also need Netcode for Game Objects to use these samples. Complete the [Hello World Getting Started](https://docs-multiplayer.unity3d.com/netcode/current/tutorials/helloworld) to prepare your environment.
```
Comment (fernando-cortez) [inline README.md:91]:
```suggestion
**PLEASE NOTE:** You will also need Netcode for Game Objects to use these samples. Complete the [Hello World tutorial](https://docs-multiplayer.unity3d.com/netcode/current/tutorials/helloworld) to prepare your environment.
```

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**